src/components/NotesMindMapView/NotesMindMapViewContainer.js

```diff
@@ -16,6 +17,7 @@ const mapStateToProps = ({ notesMindMap: { selectedNote } }) => {
 const mapDispatchToProps = () => ({
   requestNoteText: data => action(REQUEST_NOTE_TEXT_ACTION, data),
   changeSelectedNote: data => action(CHANGE_SELECTED_NOTE_ACTION, data),
+  changeBrainVisNetworkNote: data => action(CHANGE_NOTE_BVN_ACTION, data),
```

Comment:
I don't understand what does it mean to change a brain vis network note.

Also "Brain" is the old name of the app :) It can be omitted now. LIke `changeVisNetworkNote`.

Reply:
And same for the event name - `CHANGE_NOTE_BVN_ACTION` B is for Brain which should be removed.

And I don't understand what this action means and does as well.

Reply:
> I don't understand what does it mean to change a brain vis network note.

That action to say vis network that it should update note.
Probably will be simplified when all project become React.

Reply:
> Also "Brain" is the old name of the app :) It can be omitted now. LIke `changeVisNetworkNote`.

fixed